README.md

```diff
@@ -9,6 +9,8 @@ Most of our communication is done on gitter.im in the [opentelemetry-php](https:
 Our meetings are held weekly on zoom on Wednesdays at 10:30am PST / 1:30pm EST.  
 A Google calendar invite with the included zoom link can be found [here](https://calendar.google.com/event?action=TEMPLATE&tmeid=NmkyMTM1cWtlNGlmODdrMTZsZW9qa2tzdDRfMjAyMDAyMTlUMTgzMDAwWiBnb29nbGUuY29tX2I3OWUzZTkwajdiYnNhMm4ycDVhbjVsZjYwQGc&tmsrc=google.com_b79e3e90j7bbsa2n2p5an5lf60%40group.calendar.google.com&scp=ALL)
 
+Our open issues can all be found in the [github issues tab](https://github.com/open-telemetry/opentelemetry-php/issues).  Feel free to reach out in gitter if you have any additional questions about these issues; we are always happy to talk through implementation details.
```

Comment:
should we add here a link to our gitter channel?

Reply:
There is one on line 2 of this file; it's a badge.  Do you think we should change the way we belay this information?

Reply:
Sorry, I missed that.